android/java/org/chromium/chrome/browser/download/settings/BraveDownloadSettings.java

```diff
@@ -66,6 +87,13 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
                             BravePreferenceKeys.BRAVE_DOWNLOADS_AUTOMATICALLY_OPEN_WHEN_POSSIBLE,
                             (boolean) newValue)
                     .apply();
+        } else if (PREF_DOWNLOAD_PROGRESS_NOTIFICATION_BUBBLE.equals(preference.getKey())) {
+            ContextUtils.getAppSharedPreferences()
```

Comment:
can we use SharedPreferenceManager here ?

Reply:
Not sure about this. we are already using `ContextUtils.getAppSharedPreferences()` in download setting's another item